.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
```

Comment:
The .vscode/ folder shouldn't get added to the repo. To make it easier to push/pull changes without having to worry about this getting added, you can add the following to the `.gitignore` file:

`.vscode`